camp/api/v1/urls.py

```diff
@@ -16,7 +16,7 @@
     path('methane/<int:methane_id>/upload/', MethaneDataUpload.as_view(), name='methane-data-upload'),
     path('methane/<int:methane_id>/data/', MethaneData.as_view(), name='methane-data'),
     path('marker.png', endpoints.MapMarker.as_view(), name='marker'),
-
+    path('hms_smoke/', include('camp.api.v1.hms_smoke.urls', namespace='hms_smoke')),
```

Comment:
Let's roll with `hms-smoke` with a hyphen, just for consistency in our API paths.